recipe/meta.yaml

```
@@ -16,26 +16,34 @@ build:
 
 requirements:
   build:
+    - {{ stdlib("c") }}
     - {{ compiler('cxx') }}
     - cmake
     - make
-  run_constrained:
-    - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx]
+  host:
+    - zlib
+    - bzip2
 
 test:
   source_files:
     - source/test
   requires:
+    - {{ stdlib("c") }}
     - {{ compiler('cxx') }}
     - {{ compiler('fortran') }}
+    - coreutils  # [osx]
   commands:
-    # Workaround for lack of CONDA_BUILD_SYSROOT variable in staged recipes
-    - export CONDA_BUILD_SYSROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk  # [osx]
+    - # Ensure 'g' prefixed Unix commands available for macOS
```

```suggestion
    # Ensure 'g' prefixed Unix commands available for macOS
```

@Kwaizer The `-` in front of the comment is causing an error as it is getting an empty command

```

```

which fails everything.